contracts/contracts/harvest/Harvester.sol

```diff
@@ -171,6 +192,8 @@ contract Harvester is Governable {
             token.safeApprove(oldUniswapAddress, 0);
         }
 
+        token.safeApprove(uniswapV3Addr, type(uint256).max);
```

Comment:
If we want to be super clean we should call approve only when current _v3Fee == 0 (or not set) and the current one is not 0.